deployment/addons/helm-argo/values.yaml

```diff
@@ -0,0 +1,3 @@
+configs:
```

Comment:
Did you decide against adding ingress in the values.yaml as we discussed as opposed to the separate ingress object?

Reply:
Apologies for the confusion, have corrected it now.